Godeps/_workspace/src/github.com/jbenet/go-peerstream/conn.go

```diff
@@ -144,7 +143,7 @@ func ConnInConns(c1 *Conn, conns []*Conn) bool {
 
 // addConn is the internal version of AddConn. we need the server bool
 // as spdystream requires it.
-func (s *Swarm) addConn(netConn net.Conn, server bool) (*Conn, error) {
+func (s *Swarm) addConn(netConn net.Conn, isServer bool) (*Conn, error) {
```

Comment:
its for SPDY. 

SPDY keeps a notion of "server vs client", which means only that stream IDs of streams opened by that side are odd or even. (that way it's deterministic without having to exchange any information, zero RTT.)